IFC4x3/Properties/l/Length_0Cx6_hSdn2hR4RarQdjYBy/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Length_0Cx6_hSdn2hR4RarQdjYBy" Name="Length" UniqueId="0cec6fab-727c-42ad-b11b-9356a7b622fc" PrimaryDataType="IfcPositiveLengthMeasure" />
```

Comment:
I think the name Length is taken.

Reply:
I suggested this to Juha who commented in the following way: 
"I’m rather sceptical about the idea of defining attributes/properties taken out of their context, tends to become data rather than information as you clean the semantic meaning out of it.

If the problem for harmonizers is that the same property name appears in different PSets with different definitions, the appropriate solution in my mind would be more descriptive  naming rather that generalizing the definitions.

This length could be called NominalLength (the same in Psets having the same definition: course, kerbstone, precast kerb, transition section)? All these (and perhaps the couple of Psets that already have NominalLength) would need a small update in their definitions (none of the original ones seemed perfect, that’s why I proposed my own). 

The same could be done with thickness: NominalThickness with generic definition, the transition specific explanation could then be in NominalThicknessEnd".

Shall I change names? Have we used "Nominal..." as a prefix now in other places for Pset property names?